src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -158,6 +183,11 @@ class TezosNodeOperator(val node: TezosRPCInterface, val network: String, batchC
   def getAccountsForBlocks(accountsBlocksIndex: Map[AccountId, BlockReference]): PaginatedAccountResults = {
     import TezosTypes.Syntax._
 
+    val reverseIndex =
+      accountsBlocksIndex.groupBy{case (id, blockRef) => blockRef._1}
```

Comment:
can you pattern match also the `blockRef`? `_.1` is not very descriptive

Reply:
done